src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -304,7 +304,7 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
         asset,
         to: quote.inbound_address,
         from: maybeFromUTXOAccountAddress,
-        sendMax: Boolean(!isUtxoChainId(chainId) && state?.deposit.sendMax),
+        sendMax: Boolean(state?.deposit.sendMax),
```

Comment:
`split` algorithm would be used which shouldn't make any different since we're dealing with a single UTXO, but better use the actual chain-adapters heuristics instead of a poor man's `sendMax`